inlang/source-code/editor/src/pages/@host/@owner/@repository/State.tsx

```diff
@@ -459,11 +518,40 @@ export function EditorStateProvider(props: { children: JSXElement }) {
 			}
 		},
 		async () => {
-			const repoMeta = await repo()?.getMeta()
+			const repoMeta = await repo()!.getMeta()
+			if ("error" in repoMeta) {
+				setLixErrors([repoMeta.error, ...lixErrors()])
+			}
 			return repoMeta
 		}
 	)
 
+	const isForkSyncDisabled = () =>
```

Comment:
@martin-lysk  you moved this function which makes diffing impossible in the pr, could you move it back or was there a reason?

Reply:
it removed the repo argument and make it part of the state it self - i needed to move it at the position to have access to repo()

Reply:
double checked the function it self has not changed